main/src/library/Map.flix

```diff
@@ -653,4 +653,11 @@ namespace Map {
     pub def toIterator(m: Map[a, b]): Iterator[(a, b)] & Impure =
         List.toIterator(Map.query(_ -> EqualTo, m))
 
+    ///
+    /// Returns a copy of `m` as a `DelayMap`.
```

Comment:
We don't really use the terminology of "a copy`. I think we should just say that it returns `m` as a DelayMap.